src/deprecated/ActionList/index.ts

```diff
@@ -0,0 +1,21 @@
+import {List} from './List'
+import {Group} from './Group'
+import {Item} from './Item'
+import {Divider} from './Divider'
+export type {ListProps as ActionListProps} from './List'
+export type {GroupProps} from './Group'
+export type {ItemProps} from './Item'
+
+/**
+ * Collection of list-related components.
+ */
+export const ActionList = Object.assign(List, {
```

Comment:
Can we add a `@deprecated` JS-Doc annotation here. [Like the others...](https://github.com/primer/react/blob/next/v35.0.0/src/SelectMenu/SelectMenu.tsx#L112)

Reply:
Done!